low-level/files/iec_mapping/sspl

```diff
@@ -0,0 +1,10 @@
+0050010001,Ipmitool,Ipmitool command execution error
```

Comment:
What is the use of this file?

Reply:
this file we are referring in iem sensor to fetch event info, like - component-id, module-id, even-id.